changelogs/unreleased/4007-ywk253100

```diff
@@ -0,0 +1 @@
+Wait the namespace deletion completed before removing the CRDs when uninstalling the velero. This removes the `--wait` flag of the `uninstall` command
```

Comment:
What if the user using Velero CLI as a script.
Remove the flag probably makes the script failed.

Reply:
Yes, this may break the scripts using the velero cli. But the `--wait` option is indeed useless after the changes

Reply:
cc @dsu-igeek

Reply:
I think probably we could ignore the `--wait` flag (add the deprecated message in the flag help message) rather than directly removes the flag because it'll cause the command error out especially for the user running this command by a script tool.

Reply:
@jenting I would be OK with that. `--wait` would still result in the desired waiting, but since we're always waiting, it just becomes a no-op.

Reply:
Yes, so I think we could keep the flag `--wait` but ignores it since Velero always waiting.

Reply:
@jenting @sseago @dsu-igeek I have updated the PR to make the `--wait` as a no-op, please take a look again